js/src/components/modals/CountrySelector.js

```diff
@@ -0,0 +1,44 @@
+/* External dependencies */
+import React from "react";
+import PropTypes from "prop-types";
+import { Fragment } from "@wordpress/element";
+
+/**
+ * The Country selector component.
+ */
+class CountrySelector extends React.Component {
+	/**
+	 * Constructs the country selector.
```

Comment:
Missing `@summary`.

Reply:
This will be done in a separate issue on Jira